### PR TITLE
Fixed the authenticity of 52 Programming Problems by Tamim Shahriar Subeen

### PR DESCRIPTION
**Pull request by AzmainAhnaf**
## What does this PR do?
Add resource(s)

### How do we know it's really free?
The source is the author's own website

### For book lists, is it a book? For course lists, is it a course? etc.
It's a book with tie up to an online judge

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-bn.md [hunk 1]
```diff
@@ -64,6 +64,11 @@
 * [SL3 Framework - Code For Brain](https://web.archive.org/web/20201024204437/https://sl3.app) - Stack Learners *(:card_file_box: archived)*
 
 
+### Problem SOlving
```
Comment (eshellman) [inline books/free-programming-books-bn.md:67]:
```suggestion
### Problem Sets
```

File: books/free-programming-books-bn.md [hunk 2]
```diff
@@ -64,6 +64,11 @@
...
+
+* [৫২ টি প্রোগ্রামিং সমস্যা](http://cpbook.subeen.com/p/blog-page_11.html) - Tamim Shahriar Subeen (HTML)
```
Comment (eshellman) [inline books/free-programming-books-bn.md:69]: This is blog page 11? Is there a link for the whole work?

**Review by eshellman — CHANGES_REQUESTED** [review]

File: books/free-programming-books-bn.md (hunk 2 above)
Comment (AzmainAhnaf) [reply books/free-programming-books-bn.md:69]: as per the link, the page might be called the blog page 11 but it is indeed the only page that contains everything from the book "52 Programming Problems".

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Review by AzmainAhnaf — COMMENTED** [review]
Change Reviewed

**Pull request merged**